tests/models/test_hf_model.py

```diff
@@ -6,7 +6,7 @@
 import tempfile
```

Comment:
missing tests:
- peft/task type conversion to upper case
- error for unsupported peft type